### PR TITLE
Test case for user story 258

### PR DESCRIPTION
**Pull request by carwelsh**
# Issue

#258

## Description

_FILL IN DETAILS HERE_

## Checklist for Pull Requests

- [ ] Supplied as many details as possible on this change
- [ ] Included the link to the associated issue in the Issue section above
- [ ] The code conforms to the general [development guidelines](https://github.com/SAP/guided-answers-extension/blob/main/docs/developer-guide.md).
- [ ] The code has **unit tests** where applicable and is easily unit-testable
- [ ] This branch is appropriately named for the associated issue

### CODE REVIEW

File: docs/testcases/View Guided Answers Content_V3.md [hunk 1]
```diff
@@ -59,7 +66,51 @@ Guided Answers extension opens in view port
...
+1. Open VSCode
+2. Select Code in VSCode Nav bar
+3. Select Settings in drop down menu
+4. Click Settings in menu 
+5. Enter 'Guided Answers' in the setting search input field 
+6. Check/Enable 'SAP>UX>Guided Answer:Open in New Tab 
+7. Close Settings
+8. Open the 'Command Palette'
+9. Enter 'SAP: Open Guided Answers'
+10. Select 'SAP: Open Guided Answers' ( See expected result 1 )
```
Comment (ritarora) [inline docs/testcases/View Guided Answers Content_V3.md:85]: settings need to be set after this step to make it open in a new tab after step 13

File: docs/testcases/View Guided Answers Content_V3.md [hunk 2]
```diff
@@ -59,7 +66,51 @@ Guided Answers extension opens in view port
...
+2. Click on menu icon in side tool bar
+3. Select File
+4. Select Preferences
+6. Click Settings in menu 
+7. Enter 'Guided Answers' in the setting search input field 
+8. Check/Enable 'SAP>UX>Guided Answer:Open in New Tab 
+9. Close Settings
+10. Open the 'Command Palette'
+11. Enter 'SAP: Open Guided Answers'
+12. Select 'SAP: Open Guided Answers' ( See expected result 1 )
```
Comment (ritarora) [inline docs/testcases/View Guided Answers Content_V3.md:102]: settings need to be set after this step to make it open in a new tab after step 13

**Review by carwelsh — COMMENTED** [review]
@ritarora - Added your changes and tidy it up

**Review by ritarora — APPROVED** [review]

**Review by carwelsh — COMMENTED** [review]
@Klaus-Keller - I forgot to merge this test case update. Ritika gave it the ok a while back but I am unable to merge it. Is it possible to get this PR again or do I have to create the PR and changes again?

**Pull request closed without merging**